examples/kbleds.c

```diff
@@ -21,6 +21,10 @@ static char kbledstatus = 0;
 #define ALL_LEDS_ON 0x07
 #define RESTORE_LEDS 0xFF
 
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 15, 0)
```

Comment:
We tend to drop the support of kernel versions prior to 4.15. So, you can rewrite the examples without conditional builds.